pkg/tiller/release_server.go

```diff
@@ -59,6 +59,8 @@ var (
 	errMissingRelease = errors.New("no release provided")
 	// errInvalidRevision indicates that an invalid release revision number was provided.
 	errInvalidRevision = errors.New("invalid release revision")
+	//errInvalidName indicates that an invalid release name was provided
+	errInvalidName = errors.New("invalid release name")
```

Comment:
@bacongobbler  I was modified the return message as which you suggested. Can you help me to review the commit again. So many thanks.